src/test/java/be/jidoka/jdk/keycloak/admin/service/KeycloakUserAdminServiceSendUserActionEmailsTest.java

```diff
@@ -0,0 +1,40 @@
+package be.jidoka.jdk.keycloak.admin.service;
+
+import be.jidoka.jdk.keycloak.admin.IntegrationTest;
+import be.jidoka.jdk.keycloak.admin.domain.SendUserActionEmailRequest;
+import be.jidoka.jdk.keycloak.admin.domain.SendUserActionEmailRequestBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.InternalServerErrorException;
+import java.util.Set;
+
+import static be.jidoka.jdk.keycloak.admin.domain.CreateUserFixture.bertenBoedhoe;
+import static be.jidoka.jdk.keycloak.admin.domain.UserAction.UPDATE_PASSWORD;
+import static be.jidoka.jdk.keycloak.admin.domain.UserAction.VERIFY_EMAIL;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class KeycloakUserAdminServiceSendUserActionEmailsTest extends IntegrationTest {
+
+	@Autowired
+	private KeycloakUserAdminService keycloakUserAdminService;
```

Comment:
Spin up an email container in this test. Use https://hub.docker.com/r/mailhog/mailhog for this.

Reply:
Done.